lib/winapi/libloaderapi.c

```diff
@@ -37,7 +37,7 @@ static PIMAGE_EXPORT_DIRECTORY find_edataxb (void)
     PXBE_SECTION_HEADER section_header_addr = CURRENT_XBE_HEADER->PointerToSectionTable;
 
     for (DWORD i = 0; i < num_sections; i++) {
-        if (memcmp(section_header_addr[i].SectionName, ".edataxb", 8) == 0) {
+        if (strncmp(section_header_addr[i].SectionName, ".edata", 8) == 0) {
```

Comment:
> The function name is misleading now.

Fixed.

> In the future, we'll likely want a function to find a generic XBE section.

True. I haven't looked into it much yet, but if there is a winapi solution for this, I'll likely prefer that (and modify this code to use it when it lands), otherwise, I'd suggest such functionality goes into the nxdk lib, and then it probably makes sense to keep this piece of code separate. For now this code only exists to not clutter `GetProcAddress` with the section search.